tests/send-to-helix-workload-tests.targets

```diff
@@ -51,18 +64,21 @@
            Text="Could not find dotnet-coverage tool. %24(_DotNetCoverageToolPath)=$(_DotNetCoverageToolPath)" />
 
     <ItemGroup>
+      <_E2ETestScenarios Include="basicservices" />
+      <_E2ETestScenarios Include="cosmos" />
```

Comment:
? I thought we didn't run this in CI.

or is the intention to start running it now?